.gitignore

```diff
@@ -7,3 +7,4 @@ venv/
 .venv/
 .pytest_cache/
 **__pycache__/
+db.sqlite3
```

Comment:
There is an extra space at the end of the `db.sqlite3` line. This might prevent the file from being ignored properly. Consider removing the space to ensure it functions as expected.